src/stitches.config.ts

```diff
@@ -239,6 +231,14 @@ export const {
     transitions: {
       quick: 'all 0.2s',
     },
+    toastify: {
+      'toast-width': '80% !important',
+      'text-color-error': newFigmaColors.error6 + ' !important',
+      'text-color-success': newFigmaColors.success6 + ' !important',
```

Comment:
we could also do some of these via direct CSS in useToast or ToastContainer... so maybe its better to do it all in one place? But the variable changing is a bit lighter weight.

Reply:
I like having the stitches variables wherever possible.  I'm really surprised we need the `!important`s in there.  What's the point of them using css variables if they can't be easily overwritten.  Is there a cascade issue going on, like maybe their css needs to be imported at a different spot?